clientv3/cluster.go

```diff
@@ -17,10 +17,11 @@ package clientv3
 import (
 	"context"
 
```

Comment:
the import format needs to be fixed.

Reply:
Sure. FYI, it is not in the final version. "error" is imported here temporarily because the MemberPromote API returns error "not implemented". The "error" import is removed when the actual implementation is done.